src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -86,9 +74,10 @@ form {
 
 .body {
   margin: 0 32px;
-  padding-right: 24px;
+  padding: 32px;
   flex-grow: 1;
   overflow-y: auto;
+  color: var(--kd-color-text-level-secondary);
```

Comment:
Might want to double-check on this, as it fails contrast accessibility.

Reply:
i'm not seeing the accessibility errors:
<img width="1023" alt="Screenshot 2024-12-10 at 9 46 41 AM" src="https://github.com/user-attachments/assets/37292d7a-6b33-436e-994e-b651673b598c">

<img width="1370" alt="Screenshot 2024-12-10 at 9 47 14 AM" src="https://github.com/user-attachments/assets/71de3485-2ac3-4f58-a8e1-8dcf758c88b6">

Reply:
Ah, it was in light mode

Reply:
i see, you need to toggle light or dark with the drawer open for the violations to appear. got them down to just button violations now that should be sorted with the updated button component

<img width="1181" alt="Screenshot 2024-12-10 at 10 09 28 AM" src="https://github.com/user-attachments/assets/1731b2d6-c958-484a-a42a-a9132d0bca69">